providers/protondb/protondb.go

```diff
@@ -0,0 +1,25 @@
+package protondb
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/zquestz/s/providers"
+)
+
+func init() {
+	providers.AddProvider("protondb", &Provider{})
+}
+
+// Provider merely implements the Provider interface.
+type Provider struct{}
+
+// BuildURI generates a search URL for ProtonDB.
+func (p *Provider) BuildURI(q string) string {
+	return fmt.Sprintf("https://www.protondb.com/search?q=%s", url.QueryEscape(q))
+}
+
+// Tags returns the tags relevant to this provider.
+func (p *Provider) Tags() []string {
+	return []string{"search"}
```

Comment:
I would remove the "search" tag. This means general search, and most people won't want to check the protondb for general search content.

search (bing, brave, duckduckgo, ecosia, gibiru, google, presearch, qwant, yahoo, yandex)

Reply:
How about `games`?